Algorithm.Framework/Portfolio/EqualWeightingPortfolioConstructionModel.cs

```diff
@@ -60,6 +60,11 @@ public override IEnumerable<IPortfolioTarget> CreateTargets(QCAlgorithmFramework
                 .Where(x => x.CloseTimeUtc > algorithm.UtcTime)
                 .Select(x => x.Symbol).Distinct().ToList();
 
+            if (symbols.Count == 0)
+            {
+                return targets;
```

Comment:
Looking at this I noticed https://github.com/QuantConnect/Lean/pull/2266/files#diff-8b46b0f4f1c7d13b4660c2f0121a289cR55 -- I think he should returning `targets` as well, yes? Otherwise we may drop removed symbols.